sky/tools/create_xcframework.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import argparse
+import errno
+import os
+import shutil
+import subprocess
+import sys
+
+def main():
+  parser = argparse.ArgumentParser(
+      description='Creates an XCFramework consisting of the specified universal frameworks')
+
+  parser.add_argument('--frameworks',
+    nargs='+', help='The framework paths used to create the XCFramework.', required=True)
+  parser.add_argument('--name', help='Name of the XCFramework', type=str, required=True)
+  parser.add_argument('--location', help='Output directory', type=str, required=True)
+
+  args = parser.parse_args()
```

Comment:
Does this automatically sys.exit if someone uses `--help`?

Reply:
Yup:
```
$ python sky/tools/create_xcframework.py --help
usage: create_xcframework.py [-h] --frameworks FRAMEWORKS [FRAMEWORKS ...]
                             --name NAME --location LOCATION

Creates an XCFramework consisting of the specified universal frameworks

optional arguments:
  -h, --help            show this help message and exit
  --frameworks FRAMEWORKS [FRAMEWORKS ...]
                        The framework paths used to create the XCFramework.
  --name NAME           Name of the XCFramework
  --location LOCATION   Output directory
```